MdeModulePkg/Bus/Pci/PciHostBridgeDxe/PciHostResource.h

```diff
@@ -24,6 +24,15 @@ typedef enum {
   TypeMax
 } PCI_RESOURCE_TYPE;
 
+// MU_CHANGE begin
+// Only x86_64 has IO resource
+#if defined(__x86_64__)

```

Comment:
Can we use the `MDE_CPU_*` to be EFI consistent?

Reply:
Updated to MDE_CPU_X64